build.sbt

```diff
@@ -75,6 +75,11 @@ lazy val http4sDeps = libraryDependencies ++= Seq(
   Libraries.http4sCirce
 )
 
+lazy val loggingLibs = libraryDependencies ++= Seq(
+  Libraries.log4s,
+  Libraries.logbackClassic % "runtime"
```

Comment:
Does log4s need this dep?

Reply:
afaik yes, because `log4s` is just a wrapper for `slf4j` and this `serves as a simple facade or abstraction for various logging frameworks (e.g. java.util.logging, logback, log4j ...`

Reply:
Yeah. To me this means we shouldn't add a dependency on `logback`, though, since maybe the library's user is using `log4j` instead.

If you don't add a slf4j implementation to the classpath, it spits out
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

and drops logs, so it won't cause any runtime crashes.

We maybe should add a note to the README / setup instructions mentioning that we use log4s/slf4j.

Reply:
@hrhino :+1: You are right